fiftyone/operators/builtin.py

```diff
@@ -1021,9 +1028,34 @@ def resolve_input(self, ctx):
                 }
             )
 
+        categorical_field_types = (
+            fo.StringField,
+            fo.BooleanField,
+            fo.ObjectIdField,
+        )
+        numeric_field_types = (
+            fo.FloatField,
+            fo.IntField,
+            fo.DateField,
+            fo.DateTimeField,
```

Comment:
`FrameNumberField` could be apply here. I only note it because it came up in the lightning backend. Reported by @minhtuev 
* https://github.com/voxel51/fiftyone/pull/5093

Reply:
In the backend we have the luxury of using `isinstance`, which understands that `FrameNumberField` is a subclass of `IntField`. So we're all set 😎